### PR TITLE
fix(costmodels): update error empty states to platforms

### PR DESCRIPTION
**Pull request by boaz0**
closes COST 653

Screenshot_2020-10-31 Cost Management(2)

Screenshot_2020-10-31 Cost Management(1)

Screenshot_2020-10-31 Cost Management

### CODE REVIEW

**Review by dlabrecq — APPROVED** [review]

**Comment by nlcwong** [comment]
Here are examples of how the full page error state would look like for the Assign sources wizard step AND modal dialog when a user wants to add when editing a cost model: [https://marvelapp.com/prototype/9404edb/screen/74151760](https://marvelapp.com/prototype/9404edb/screen/74151760).

I'm not sure what the difference between the second and last screenshot from above. Are there different errors happening in the last two screenshots?

**Comment by boaz0** [comment]
Thanks @nlcwong 
I updated the other two empty states accordingly

Screenshot_2020-11-04 Cost Management(1)

Screenshot_2020-11-04 Cost Management

**Comment by nlcwong** [comment]
The wording is not quite right and we're missing a link in the sentence.  The wording (including the error title) are not exact so they will be contextualized depending on where this error message is appearing in.  I've updated the mocks a bit.

Here is the mock for the Wizard Step: https://marvelapp.com/prototype/9404edb/screen/74151760
Here is the mock for the dialog: https://marvelapp.com/prototype/9404edb/screen/74151761

I've edited the wording a bit to better reflect the wizard or dialog that they are on. The button text are different too as they are contextual to where the user is seeing this message. Please be aware the titles are contextualized as well.

**Comment by boaz0** [comment]
@nlcwong updates:

image
image

File: src/locales/en.json [hunk 1]
```diff
@@ -282,6 +282,9 @@
     "add_source": "Assign source",
     "add_source_desc": "Review and confirm your changes. Click \"Finish\" to assign sources, or \"Back\" to revise.",
     "assign_sources": "Assign sources",
+    "assign_sources_error_action_button": "Refresh this dialog",
+    "assign_sources_error_desc": "You cannot assign a source at this time. Try refreshing this page. If the problem persists, contact your organization administrator or visit our <1>status page</1> for known outages.",
```
Comment (nlcwong) [inline src/locales/en.json:286]: Replace "Try refreshing this page." with "Try refreshing this dialog."

File: src/locales/en.json [hunk 2]
```diff
@@ -292,9 +295,9 @@
     "edit_markup_action": "Edit markup",
     "edit_rate": "Edit rate",
     "empty_state": {
-      "desc": "Create a cost model to start calculating your hybrid cloud costs using custom price lists, markups, or both. Click on the \"Create cost model\" button below to begin the journey.",
+      "desc": "Create a cost model to start calculating your hybrid cloud costs using custom price lists, markups, or both. Click on the button below to begin the journey.",
       "primary_action": "Create cost model",
-      "title": "What is your hybrid cloud costing you?"
+      "title": "At what cost is your hybrid cloud costing you?"
```
Comment (nlcwong) [inline src/locales/en.json:300]: I realize that the use of the term "cost" is repeated. So just don't do this change, revert back to the old title.

File: src/locales/en.json [hunk 3]
```diff
@@ -406,10 +409,13 @@
     },
     "source": {
       "caption": "Select from the following {{ type }} sources:",
+      "error": {
+        "action": "Refresh this step",
+        "desc": "Try refreshing this page or you can skip this step (as it is optional) and assign the source to the cost model at a later time. If the problem persists, contact your organization administrator or visit our <2>status page</2> for known outages.",
```
Comment (nlcwong) [inline src/locales/en.json:414]: Replace "Try refreshing this page" with "Try refreshing this step"

**Review by nlcwong — APPROVED** [review]
The updates LGTM

**Comment by boaz0** [comment]
Thanks @dlabrecq and @nlcwong

**Pull request merged**